docs/reference/compose_restart.md

```diff
@@ -1,7 +1,7 @@
 # docker compose restart
 
 <!---MARKER_GEN_START-->
-Restart containers
+Restart service containers
```

Comment:
This change should be done directly [here](https://github.com/docker/compose/blob/def189fae1a74a9cac327781aabeaa214d6906a9/cmd/compose/restart.go#L39)